specs/001-kiro-specs-knowledge/contracts/knowledge_graph_api.yaml

```diff
@@ -0,0 +1,258 @@
+openapi: 3.0.3
```

Comment:
I like this. contracts up front will help us think through the system more concretely and really help testing/getting to MVP.